index.html

```diff
@@ -4,32 +4,50 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@700&family=Poppins:wght@700&family=Roboto:wght@700&display=swap"
       rel="stylesheet"
     />
+    
     <title>Document</title>
   </head>
   <body>
-    <div class="max-context">
+    
+   
       <header class="mobile-header">
         <a href="#">Diego Hernández</a>
-        <a href="#"><img src="images2/menu.png" alt="hamburger Icon" /></a>
+                
+      <ul class="menu-deployed">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+          <li class="icon">
+          <img src="images2/menu.png" alt="hamburger Icon" />
```

Comment:
- [ ] Please fix the hamburger icon.
- Your design 
![h](https://user-images.githubusercontent.com/63972670/166093797-56c668d1-c21d-4c2d-8a62-1734d8ab1df0.JPG)